### DISCUSSION TITLE
How to increase the max-width of docs page content?

### DISCUSSION THREAD

**Discussion by ZYJLiu** [fuma-nama/fumadocs#692] [Q&A]
Is there a way to easily adjust the max width of the `article` in DocsPage? Specifally the [max-w-[840px]](https://github.com/fuma-nama/fumadocs/blob/main/packages/ui/src/page.tsx#L96).

I tried using `containerProps` in the layout.config.tsx file like this:

```ts
  containerProps: {
    className: "[&_article]:max-w-[1000px]",
  },
```

However, this also overrides when `!tocOptions.enabled`, which I'd like to keep as the default `max-w-[1200px]`.

Is there a suggested way to modify the width for both when toc is and isn't enabled?

**Comment by fuma-nama** [comment]
Just remove your custom class name when toc is disabled
